data/sql/updates/16_08_14_02.sql

```diff
@@ -1,4 +1,4 @@
 ALTER TABLE world_db_version CHANGE COLUMN 2016_08_14_00 2016_08_14_02 bit;
 
 UPDATE gameobject_template
-SET data10='29519' WHERE entry='181597'AND entry='181598'
+SET data10='29519' WHERE entry='181597' AND '181598'
```

Comment:
@talamortis 

if you want to apply that data10 field change to both 181597 and 181598, the sql would be:

`UPDATE gameobject_template SET data10=29519 WHERE `entry`=181597 OR `entry` = 181598`

which literally means "update the table gameobject_template, by setting data10 value to 29519, to the row(s) having entry set to 181597 or to the row(s) having entry set to 181598"